### ISSUE TITLE
support custom increment value at range loops

### ISSUE THREAD

**Issue by edilson258** [edilson258/filipe#3]
The current `range` function doesn't give ability to customize the increment rate, it always increment by one.

```python
for i in range(0, 10) {
    // some code
}
```
add support for the third `range` function to be used as increment rate.

```python
for i in range(0, 10, 2) {
   // some code
}
```

**Comment by yazaldefilimone** [comment]
did in  #9

**Comment by edilson258** [comment]
Fixed at #9
